instrumentation/micrometer/micrometer-1.5/library/src/main/java/io/opentelemetry/instrumentation/micrometer/v1_5/Bridging.java

```diff
@@ -47,7 +47,7 @@ static String description(Meter.Id id) {
 
   static String baseUnit(Meter.Id id) {
     String baseUnit = id.getBaseUnit();
-    return baseUnit == null ? "1" : baseUnit;
+    return baseUnit;
```

Comment:
Nit: local var can be inlined

Reply:
turned out `null` wasn't allowed, so it's "" for a missing base unit now